src/gateways/internal-api.ts

```diff
@@ -108,10 +144,6 @@ export class InternalApiGateway {
 
       return ResponseMapper.mapEvidenceRequest(data);
     } catch (err) {
```

Comment:
is there a (possibly completely obvious) reason for only throwing the error here vs the more elaborate handling shown above and in the previous version of this function?